### DISCUSSION TITLE
Why can't I get the attribute 'pos' data from a new model trained from scratch?

### DISCUSSION THREAD

**Discussion by qingyun1988** [explosion/spaCy#8434] [Language Support]
This is my training data example which contains attribute 'pos' data and uses for training a blank model: 

`Example.from_dict(_nlp.make_doc('I love you.'), {'heads': [1, 1, 1, 1], 'deps': ['nsubj', 'ROOT', 'dobj', 'punct'],'words': ['I', 'love', 'you', '.'],'pos': ['PRON', 'VERB', 'PRON', 'PUNCT'],'tags': ['PRP', 'VBP', 'PRP', '.']})`

But I can't get the attribute 'pos' data from a new model trained from scratch, which is empty:
![image](https://user-images.githubusercontent.com/38597436/122537548-3fbd4480-d058-11eb-98dd-e2eda490250c.png)

Why? Is the 'pos' attribute data format in the data example wrong?

Is there anyone who would like to answer my question? Thank you!

**Comment by adrianeboyd** [comment]
Hi, does your model pipeline include a `morphologizer`? The `tagger` only annotates `token.tag`, not `token.pos`. The `morphologizer` annotates `token.pos` and `token.morph`.

If this isn't the case, could you post the full config and details about which version of spacy you're using?

**Reply by qingyun1988** [reply]
Thank you very much! I've solved it through your tips above.  The version of Spacy I'm using is v3.0 and the training way I am using is API, not config.

I am very clear about how to train `token.tag` and `token.pos`  on a model after you reminded me. Thank you again.

By the way, I want to consult you a question again.

This question is about resume-training a pre-trained model.

I used 100 examples that are all "so...that..." inversion sentences to resume training on the pre-trained model `'en_core_web_sm-3.0'`, but there is a  little change in terms of dependencies which is not what I expect.

For example, there is a sentence: 
So unreasonable was his price that everybody startled.

The view before resume-training:
![image](https://user-images.githubusercontent.com/38597436/122629561-7050bd00-d0f0-11eb-9adf-c81abfec0195.png)

The view I expect:
![image](https://user-images.githubusercontent.com/38597436/122629586-a3934c00-d0f0-11eb-8679-581b107d534a.png)
i.e. I expect the head of the word `'startled'` is `'unreasonable'`, not `'was'`.

But, the view after the model has trained:
![image](https://user-images.githubusercontent.com/38597436/122629714-962a9180-d0f1-11eb-9245-baf642117095.png)
The head of the word `'startled'` didn't change at all, and the dep of the word `'startled'` was replaced` 'det'` from the previous `'ccomp'` .

Why is this change? Is the number of examples used for training not enough? Or did I do something wrong?

I expect your reply for me, thank you.

**Reply by svlandeg** [reply]
It looks like your follow-up question is the same as the one from https://github.com/explosion/spaCy/discussions/8295. It's probably better to keep things in one place.

**Reply by qingyun1988** [reply]
> It looks like your follow-up question is the same as the one from #8295. It's probably better to keep things in one place.

@svlandeg There is a little difference between this question and the former one #8295. In #8295, the model I've trained on is 'lg' and in this question, the model is 'sm' which is far smaller than the 'lg' one. So, the examples I prepared should theoretically be enough for having some change, but the consequence is not the same as I expected. 

My question is, were these examples not enough or did I do something wrong? Thank you!
